spread.yaml

```diff
@@ -1397,4 +1398,26 @@ suites:
             . "$TESTSLIB"/pkgdb.sh
             distro_purge_package qemu genisoimage sshpass qemu-kvm cloud-image-utils xz-utils
 
+    # FIPS test suite
+    tests/fips/:
+        summary: FIPS enabled test suite
+        systems:
+          - ubuntu-fips-*
+        environment:
+          # TODO add snap variant
```

Comment:
Is there a card for that?

Reply:
yes, [SNAPDENG-23245](https://warthogs.atlassian.net/browse/SNAPDENG-23245)